ffac-update-location-gps/files/etc/hotplug.d/usb/10-update-location-gps

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+TTYPATH="/sys${DEVPATH}/tty"
+LOCKPATH="/tmp/update-location-gps"
+
+ENABLED=$(uci get update-location-gps.settings.enabled)
```

Comment:
As this is a local variable, I would also make it lowercase instead, so shellcheck can check its usage across the file.